src/browser/core.js

```diff
@@ -17,6 +17,32 @@ define(function(require, exports, module) {
   var render = (Component, initial, target) => {
     let current = null;
 
+    const schedule = (task) => {
+      if (!schedule.d) {
+        schedule.d = true;
+
+        // Disable anymation framing as it couses some
+        // misbehavior (see https://github.com/mozilla/browser.html/issues/82)
+        //window.requestAnimationFrame(() => {
```

Comment:
Do we mean to remove requestAnimationFrame? Should we disable it only sometimes?
We should remove this code or un-comment it.